### ISSUE TITLE
Is it possible to update multiple .json files using extra-files?

### ISSUE THREAD

**Issue by aaronware** [googleapis/release-please#1622]
Our team's projects typically have both a package.json (for our javascript, sass elements) and composer.json (for our PHP dependencies) within the root of our projects. We're trying to bump both the main "PHP" project's composer.json as well as a package.json (within our extra-files).

- What code you've already tried
Utilized the package.json within the extra files command though extra files only utilize the generic updater but this won't work due to JSON files not allowing comments

- Any error messages you're getting
No errors, the package.json is simply ignored.

I tried to do a quick github search but didn't see any projects doing this, which I thought was interesting as I didn't think we were doing anything out of the ordinary.

Any shove in the right direction would be greatly apprecaited

**Comment by chingor13** [comment]
If you're using a manifest config, you can specify a generic JSON file updater: https://github.com/googleapis/release-please/blob/main/schemas/config.json#L123-L141. This was implemented in #1332

Example:

```json
{
  "packages": {
    ".": {
      "extra-files": [
        {
          "type": "json",
          "path": "composer.json",
          "jsonpath": "$.version"
        }
      ]
    }
  }
}
```

**Comment by danielo515** [comment]
Is there any comprehensive guide about how to setup this in the github action? I find having to ping-pong through the cli and gothub action docs very confusing,  and I don't want to install any CLI just to setup my CI
